docs/_data/icons.yml

```diff
@@ -354,6 +354,8 @@
   unicode: "\e0b0"
 - name: message-error
   unicode: "\e0b1"
+- name: notification
+  unicode: "\e0b1"
 - name: comment
```

Comment:
there are some errors in this file. you cannot have 2 icons occupy the same unicode value. This needs to be redone.

Reply:
The source of this is the sap icons. I didn’t modify any name. I guess we need to address this to the team generating the names. There was another mistake i found-the last icon in the icon library is named collapse-all, even though there is another one

Reply:
I will remove the duplicate for now and maybe we can try to automate the process of including the icon library

Reply:
it is *possible* that they are using 2 names for the same icon. 🤷‍♂ 

I would check with them before removing anything.

Reply:
in their css file is written `duplicate icon with alternative class name for compatibility reasons`